examples/07_cloudflare/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "cf-typegen": "wrangler types",
     "dev": "waku dev",
-    "build": "waku build",
+    "build": "CLOUDFLARE=1 waku build",
```

Comment:
`CLOUDFLARE=1` is not necessary as this example uses `waku/adapters/cloudflare`, not `waku/adapters/default`.